checks.d/pgbouncer.py

```diff
@@ -0,0 +1,167 @@
+"""Pgbouncer check
+
+Collects metrics from the pgbouncer database.
+"""
+from checks import AgentCheck, CheckException
+from collections import OrderedDict
+
+import psycopg2 as pg
+class ShouldRestartException(Exception): pass
+
+class PgBouncer(AgentCheck):
+    """Collects metrics from pgbouncer
+    """
+    SOURCE_TYPE_NAME = 'pgbouncer'
+    RATE = AgentCheck.rate
+    GAUGE = AgentCheck.gauge
+
+    STATS_METRICS = {
+        'descriptors': [
+            ('database', 'db'),
+        ],
+        'metrics': OrderedDict([
+            ('total_requests', ('pgbouncer.stats.total_requests', RATE)),
+            ('total_received', ('pgbouncer.stats.total_received', RATE)),
```

Comment:
@DorianZaccaria what are the units for `total_received` and `total_sent`. Bytes? Connection count?

Same comment regarding the change of type.